src/main/java/Duke.java

```diff
@@ -1,10 +1,148 @@
+import java.util.Scanner;
+
 public class Duke {
-    public static void main(String[] args) {
+
+    public static Task[] tasks = new Task[100];
```

Comment:
maybe an arraylist can be used instead so that there will be no issues of exceeding the array size

Reply:
The task description in `Level-2` says that we can assume at most 100 tasks. Anyway, how to implement an arraylist?